frontend/components/panels/CollateralAuthorizationCheckPanel.stories.js

```diff
@@ -0,0 +1,87 @@
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import CollateralAuthorizationCheckPanel from './CollateralAuthorizationCheckPanel';
+import { generateFakeAuctionTransaction } from '~/helpers/generateFakeAuction';
+
+const fakeAuctionTransaction = generateFakeAuctionTransaction();
+
+const common = {
+    components: { CollateralAuthorizationCheckPanel },
+    data: () => ({
+        collateralType: fakeAuctionTransaction.collateralType,
+        authTransactionFeeETH: new BigNumber(fakeAuctionTransaction.authTransactionFeeETH),
+        walletAddress: faker.finance.ethereumAddress(),
+        isCollateralAuthorized: false,
+        isWalletAuthorized: true,
```

Comment:
You don't need to pass and to check `isWalletAuthorized` inside this component, since the panels suppose to have isolated logic and only check one thing. Therefore, please remove this prop and stories related to it

Reply:
I also removed all mentions and usages of `walletAddress`, as it was not really needed. 

If no wallet is connected, we can just pass `disabled` to the component and it would have done the same thing, which is a bit easier to manage on the long run.

Reply:
> we can just pass `disabled` to the component and it would have done the same thing

Passing disabled unfortunately wouldn't trigger `inactive` state 😞